config/webpack/loaders/devSitePackage.js

```diff
@@ -7,12 +7,23 @@ const loader = async function loader(content) {
   const callback = this.async();
 
   const json = JSON.parse(content);
+  const repoUrl = (json.repository && json.repository.url) ? (json.repository.url).match(/https.*[^.git]/i) : '';
```

Comment:
I'd like to seem some jest tests directly around this logic. I know we previously didn't have a test for this loader, but I think with this logic here it's become complex enough to warrant it.

Reply:
Added here : https://github.com/cerner/terra-dev-site/pull/243/commits/33bbee721a9d6b4b03a89595c7d87bf47b553d94